UPGRADE-7.1.md

```diff
@@ -68,6 +68,24 @@ All service classes that are loaded by `ServiceLocator::get()` must now be `publ
 # Cleanup
 # Informational
 # Changed Features
+## Cronjobs
```

Comment:
Extending TdbCmsCronjobs (instead of TCMSCronJob) must also be done.

Reply:
updated.